cli/server/server.go

```diff
@@ -65,6 +65,10 @@ func NewCommands() []cli.Command {
 			Name:  "dump",
 			Usage: "directory for storing JSON dumps",
 		},
+		cli.BoolFlag{
+			Name:  "diff",
```

Comment:
Short form?

Reply:
Do we really need it? We have `dump` already, I think it will only add confusion.

Reply:
It doesn't have to be `d`, but I really like having both long and short forms available.

Reply:
Fixed.

Reply:
Used `-d`, because `--dump` is used for debug only and can even be hidden.

Reply:
Well, no, `-d` is for `debug`. `-c` for "changeset" is also taken by `--count`. And most of the words for "difference" are starting with either `c` or `d`. So I used `k`.